relay/worker/output_parser.go

```diff
@@ -111,3 +115,15 @@ func extractTemplate(line []string, resp *messages.ExecutionResponse, req messag
 func flagJSON(line []string, resp *messages.ExecutionResponse, req messages.ExecutionRequest) {
 	resp.IsJSON = true
 }
+
+func parseAction(line []string, resp *messages.ExecutionResponse, req messages.ExecutionRequest) {
+	if len(line) < 2 {
```

Comment:
```go
package main

import (
	"fmt"
	"regexp"
)

func main() {
	r := regexp.MustCompilePOSIX("^COGCMD_ACTION:")
	text := "COGCMD_ACTION: abort"
	fmt.Printf("Parsing: %s\n", text)
	for i, v := range r.Split(text, 2) {
		fmt.Printf("text[%d] = %s\n", i, v)
	}
	text = "COGCMD_ACTION:"
	fmt.Printf("Parsing: %s\n", text)
	for i, v := range r.Split(text, 2) {
		fmt.Printf("text[%d] = %s\n", i, v)
	}

}
```

I thought the above code (analogous to how Relay currently parses command output) would produce the following output:

```
Parsing: COGCMD_ACTION: abort
text[0] = 
text[1] =  abort
Parsing: COGCMD_ACTION:
text[0] =
```

Meaning that `Split` would return an array w/2 or 1 member depending on if the string contained content past the matching regex. Turns out I was wrong at least wrt to Go's behavior. In reality the above code produces:

```
Parsing: COGCMD_ACTION: abort
text[0] =
text[1] =  abort
Parsing: COGCMD_ACTION:
text[0] =
text[1] =
```

tl;dr You found a bug :)